ci/cloudformation/auth/function/email-notification-sqs.yaml

```diff
@@ -90,6 +90,8 @@ Resources:
           Type: SQS
           Properties:
             Queue: !GetAtt EmailNotificationQueue.Arn
+            FunctionResponseTypes:
+              - ReportBatchItemFailures
```

Comment:
@Joe-Edwards-GDS I think we still need to do modifications to the terraform configuration for the Function too (AFAIK these are WIP templates while the secure pipeline migration takes place)